generators/java/spring/src/main/java/com/fern/java/spring/SpringCustomConfig.java

```diff
@@ -18,14 +18,15 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fern.java.DownloadFilesCustomConfig;
 import com.fern.java.ICustomConfig;
 import com.fern.java.immutables.StagedBuilderImmutablesStyle;
 import org.immutables.value.Value;
 
 @Value.Immutable
 @StagedBuilderImmutablesStyle
 @JsonDeserialize(as = ImmutableSpringCustomConfig.class)
-public interface SpringCustomConfig extends ICustomConfig {
+public interface SpringCustomConfig extends ICustomConfig, DownloadFilesCustomConfig {
```

Comment:
To avoid future issues, we now just have a single `SpringCustomConfig` and no `SpringDownloadFilesCustomConfig`. This makes sense because we only support downloading files for Spring